### PR TITLE
KH-212: Add missing khmer translations

### PR DESCRIPTION
**Pull request by kdaud**
Refer to [KH-212](https://mekomsolutions.atlassian.net/browse/KH-212)

### CODE REVIEW

File: configs/openmrs_config/concepts/past_medical_history.csv [hunk 1]
```diff
@@ -12,16 +12,16 @@ ea543526-f3d3-4a23-a8aa-0c0acf529d51,,,Referred to TB section,បានបញ្
...
 5d65d35a-8ccc-474a-a2ce-746a518ed6f5,,,Use of traditional medicine,ការប្រើប្រាស់ឱសថបុរាណ,Use of traditional medicine,ការប្រើប្រាស់ឱសថបុរាណ,,,Question,Coded,Yes;No,,,
-6bb3f867-70ee-4854-9408-d386c99c5990,,,History of premature heart disease or stroke in first-degree relatives,,History of premature heart disease or stroke in first-degree relatives,,,,Question,Coded,Yes;No,,,
-4d180dd2-9bc7-4ea8-b64e-5e851c9e5c7d,,,History of diabetes or kidney disease in first-degree relatives,,History of diabetes or kidney disease in first-degree relatives,,,,Question,Coded,Yes;No,,,
+6bb3f867-70ee-4854-9408-d386c99c5990,,,History of premature heart disease or stroke in first-degree relatives,"ធ្លាប់មានប្រវត្តិជំងឺបេះដូងឬគ្រោះថ្នាក់សរសៃឈាមខួរក្បាលចំពោះសាច់ញាតិបង្កើត(ជំនាន់ទីមួយ)                    ",History of premature heart disease or stroke in first-degree relatives,"ធ្លាប់មានប្រវត្តិជំងឺបេះដូងឬគ្រោះថ្នាក់សរសៃឈាមខួរក្បាលចំពោះសាច់ញាតិបង្កើត(ជំនាន់ទីមួយ)                       ",,,Question,Coded,Yes;No,,,
```
Comment (kazlaw) [inline configs/openmrs_config/concepts/past_medical_history.csv:23]: @kdaud kindly remove the space after the question in Khmer

File: configs/openmrs_config/concepts/past_medical_history.csv [hunk 2]
```diff
@@ -12,16 +12,16 @@ ea543526-f3d3-4a23-a8aa-0c0acf529d51,,,Referred to TB section,បានបញ្
...
+4d180dd2-9bc7-4ea8-b64e-5e851c9e5c7d,,,History of diabetes or kidney disease in first-degree relatives,"ធ្លាប់មានប្រវត្តិជំងឺទឹកនោមផ្អែម ឬជំងឺតម្រងនោមចំពោះសាច់ញាតិបង្កើត   (ជំនាន់ទីមួយ)   ",History of diabetes or kidney disease in first-degree relatives,"ធ្លាប់មានប្រវត្តិជំងឺទឹកនោមផ្អែម ឬជំងឺតម្រងនោមចំពោះសាច់ញាតិបង្កើត   (ជំនាន់ទីមួយ)   ",,,Question,Coded,Yes;No,,,
```
Comment (kazlaw) [inline configs/openmrs_config/concepts/past_medical_history.csv:24]: @kdaud kindly remove the space after the question in Khmer

File: configs/openmrs_config/concepts/physical_examination.csv [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-735dc19f-78f0-4f0f-861b-e2b4328521b4,,,AKA,,AKA,,,,Misc,N/A,,,,
-bb9a023b-a077-4aad-baac-021c8ed1f626,,,BKA,,BKA,,,,Misc,N/A,,,,
-81318009-e9fb-4b8a-9bf6-17082cefd19e,,,Symptom of PN-Pain,,Symptom of PN-Pain,,,,Misc,N/A,,,,
-be0c7342-12da-48c1-a247-89379f80e18d,,,Symptom of PN-Pins and Needle,,Symptom of PN-Pins and Needle,,,,Misc,N/A,,,,
-4ff6c3ea-2be8-4aac-b371-faa3bb450255,,,Symptom of PN-Numbness,,Symptom of PN-Numbness,,,,Misc,N/A,,,,
-8c001d6a-9525-4008-8d06-eebc713f5d8c,,,Symptom of PN-Burning,,Symptom of PN-Burning,,,,Misc,N/A,,,,
-0571152b-1f18-4603-9c86-d77883525e61,,,Symptom PAD-Yes,,Symptom PAD-Yes,,,,Misc,N/A,,,,
-c9cfaaf9-0f14-4805-a151-b0544cbd8870,,,Symptom of PAD-No,,Symptom of PAD-No,,,,Misc,N/A,,,,
-7640b59c-0692-4736-90a1-52a9f5f2d089,,,Pulses-Normal,,Pulses-Normal,,,,Misc,N/A,,,,
+735dc19f-78f0-4f0f-861b-e2b4328521b4,,,AKA," ការវះកាត់យកជើងខាងលើជង្គង់ចេញ ",AKA," ការវះកាត់យកជើងខាងលើជង្គង់ចេញ ",,,Misc,N/A,,,,
```
Comment (kazlaw) [inline configs/openmrs_config/concepts/physical_examination.csv:4]: @kdaud , there is a space here too

File: configs/openmrs_config/concepts/physical_examination.csv [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
+bb9a023b-a077-4aad-baac-021c8ed1f626,,,BKA,ការវះកាត់យកជើងខាងក្រោមជង្គង់ចេញ,BKA,ការវះកាត់យកជើងខាងក្រោមជង្គង់ចេញ,,,Misc,N/A,,,,
+81318009-e9fb-4b8a-9bf6-17082cefd19e,,,Symptom of PN-Pain,"រោគសញ្ញាជំងឺសរសៃប្រសាទឈឺ ",Symptom of PN-Pain,"រោគសញ្ញាជំងឺសរសៃប្រសាទឈឺ ",,,Misc,N/A,,,,
```
Comment (kazlaw) [inline configs/openmrs_config/concepts/physical_examination.csv:6]: @kdaud here too before and after

File: configs/openmrs_config/concepts/physical_examination.csv [hunk 5]
```diff
@@ -1,20 +1,20 @@
...
+7640b59c-0692-4736-90a1-52a9f5f2d089,,,Pulses-Normal,ជីពចរជើងធម្មតា,Pulses-Normal,ជីពចរជើងធម្មតា,,,Misc,N/A,,,,
 bbf9ee15-e3df-4026-825f-e17325027795,,,Pulse-Abnormal,,Pulse-Abnormal,,,,Misc,N/A,,,,
-eceb2426-dd37-490d-b4f0-711c5f76060b,,,Pulse-Normal-Detectable,,Pulse-Normal-Detectable,,,,Misc,N/A,,,,
-163747AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,Absent,,Absent,,,,Misc,N/A,,,,
-9661c8c0-d5d6-4b80-8043-fa1d5fa0ec94,,,Monofilament-Normal,,Monofilament-Normal,,,,Misc,N/A,,,,
-b5aead5b-e0ed-4462-8d38-44cc0ab63dfc,,,Monofilament-Abnormal,,Monofilament-Abnormal,,,,Misc,N/A,,,,
+eceb2426-dd37-490d-b4f0-711c5f76060b,,,Pulse-Normal-Detectable,ជីពចរជើង - ធម្មតា -ស្ទាបដឹង,Pulse-Normal-Detectable,ជីពចរជើង - ធម្មតា -ស្ទាបដឹង,,,Misc,N/A,,,,
+163747AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,Absent,គ្មានជីពចរ ស្ទាបអត់ដឹង,Absent,គ្មានជីពចរ ស្ទាបអត់ដឹង,,,Misc,N/A,,,,
+9661c8c0-d5d6-4b80-8043-fa1d5fa0ec94,,,Monofilament-Normal,"Monofilament ធម្មតា ",Monofilament-Normal,"Monofilament ធម្មតា ",,,Misc,N/A,,,,
+b5aead5b-e0ed-4462-8d38-44cc0ab63dfc,,,Monofilament-Abnormal,"Monofilament មិនធម្មតា ",Monofilament-Abnormal,"Monofilament មិនធម្មតា ",,,Misc,N/A,,,,
```
Comment (kazlaw) [inline configs/openmrs_config/concepts/physical_examination.csv:17]: @kdaud and here as well

File: configs/openmrs_config/concepts/physical_examination.csv [hunk 6]
```diff
@@ -24,12 +24,12 @@ dd16ec73-fa62-431b-bbd5-4e3a04d93cfc,,,Left Foot Exam,ធ្វើតេស្
...
-4f09f4b9-d1bd-43b3-8487-bc138e62a6e7,,,Blurred vision,,Blurred vision,,,,Misc,N/A,,,,
-d62ed944-515d-4d61-97ed-39b3871954f7,,,Vision impairment,,Vision impairment,,,,Misc,N/A,,,,
-476d87b3-9e19-499d-8d3a-97501b176f99,,,Cataract,,Cataract,,,,Misc,N/A,,,,
-91d796f1-33dc-427c-a678-61bf12e52afd,,,Macular Oedema,,Macular Oedema,,,,Misc,N/A,,,,
-7ffd8f4f-5681-4f6f-ab79-b8499d720263,,,NPR,,NPR,,,,Misc,N/A,,,,
-e4cb0610-3d07-4e28-a321-3b376e18e4db,,,PR,,PR,,,,Misc,N/A,,,,
+4f09f4b9-d1bd-43b3-8487-bc138e62a6e7,,,Blurred vision,ព្រិលភ្នែក,Blurred vision,ព្រិលភ្នែក,,,Misc,N/A,,,,
+d62ed944-515d-4d61-97ed-39b3871954f7,,,Vision impairment,ពិការភ្នែក,Vision impairment,ពិការភ្នែក,,,Misc,N/A,,,,
+476d87b3-9e19-499d-8d3a-97501b176f99,,,Cataract,ជំងឺភ្នែកឡើងបាយ,Cataract,ជំងឺភ្នែកឡើងបាយ,,,Misc,N/A,,,,
+91d796f1-33dc-427c-a678-61bf12e52afd,,,Macular Oedema,"ម៉ាគុលឡា អឺដែម៉ា ",Macular Oedema,"ម៉ាគុលឡា អឺដែម៉ា ",,,Misc,N/A,,,,
```
Comment (kazlaw) [inline configs/openmrs_config/concepts/physical_examination.csv:30]: here too @kdaud

File: configs/openmrs_config/concepts/physical_examination.csv [hunk 7]
```diff
@@ -1,20 +1,20 @@
...
+81318009-e9fb-4b8a-9bf6-17082cefd19e,,,Symptom of PN-Pain,រោគសញ្ញាជំងឺសរសៃប្រសាទឈឺ,Symptom of PN-Pain,រោគសញ្ញាជំងឺសរសៃប្រសាទឈឺ,,,Misc,N/A,,,,
+be0c7342-12da-48c1-a247-89379f80e18d,,,Symptom of PN-Pins and Needle,រោគសញ្ញាជំងឺសរសៃប្រសាទឈឺដូចម្ជុលចាក់,Symptom of PN-Pins and Needle,រោគសញ្ញាជំងឺសរសៃប្រសាទឈឺដូចម្ជុលចាក់,,,Misc,N/A,,,,
+4ff6c3ea-2be8-4aac-b371-faa3bb450255,,,Symptom of PN-Numbness,រោគសញ្ញាជំងឺសរសៃប្រសាទស្ពឹកៗ,Symptom of PN-Numbness,រោគសញ្ញាជំងឺសរសៃប្រសាទស្ពឹកៗ,,,Misc,N/A,,,,
+8c001d6a-9525-4008-8d06-eebc713f5d8c,,,Symptom of PN-Burning,រោគសញ្ញាជំងឺសរសៃប្រសាទក្តៅដូចរលាក,Symptom of PN-Burning,រោគសញ្ញាជំងឺសរសៃប្រសាទក្តៅដូចរលាក,,,Misc,N/A,,,,
+0571152b-1f18-4603-9c86-d77883525e61,,,Symptom PAD-Yes,មានឈឺពេលសម្រាក,Symptom PAD-Yes,មានឈឺពេលសម្រាក,,,Misc,N/A,,,,
+c9cfaaf9-0f14-4805-a151-b0544cbd8870,,,Symptom of PAD-No,អត់មានឈឺពេលសម្រាក,Symptom of PAD-No,អត់មានឈឺពេលសម្រាក,,,Misc,N/A,,,,
+7640b59c-0692-4736-90a1-52a9f5f2d089,,,Pulses-Normal,ជីពចរជើងធម្មតា,Pulses-Normal,ជីពចរជើងធម្មតា,,,Misc,N/A,,,,
 bbf9ee15-e3df-4026-825f-e17325027795,,,Pulse-Abnormal,,Pulse-Abnormal,,,,Misc,N/A,,,,
```
Comment (kazlaw) [inline configs/openmrs_config/concepts/physical_examination.csv:13]: @kdaud this is missing please add it from [here](https://docs.google.com/spreadsheets/d/13PHkzorfTmYRjuIJv3fCml3zBAuDba6Mgdpu4H7cAg8/edit#gid=0&range=188:188)

`Pulse Abnormal`

Comment (kazlaw) [reply configs/openmrs_config/concepts/physical_examination.csv:13]: ជីពចរជើងធម្មតា

**Comment by kazlaw** [comment]
These two `Renal Test` and `Other Tests` please add the translations in this file [here](https://github.com/openmrs/ozone-distro-cambodia/blob/main/configs/openmrs_config/ampathformstranslations/NCD%20Consultation_translations_km.json) corresponding to the `NCD Consultation form`

**Comment by kazlaw** [comment]
km-3

**Review by kazlaw — COMMENTED** [review]
There some issues to kindly address @kdaud

**Comment by kdaud** [comment]
@kazlaw, have added the missing translations that have been recently provided.

**Review by kazlaw — APPROVED** [review]
Looks good

**Pull request merged**